server/src/service-layer/controllers/AdminController.ts

```diff
@@ -692,4 +694,20 @@ export class AdminController extends Controller {
       }
     }
   }
+
+  /**
+   * Endpoint for admin to create a new event
+   */
+  @SuccessResponse("201", "Created Event")
+  @Post("events/create")
+  public async createNewEvent(@Body() body: CreateEventBody) {
+    try {
+      const eventService = new EventService()
+      await eventService.createEvent(body.data)
+
+      this.setStatus(201)
+    } catch {
+      this.setStatus(500)
+    }
+  }
```

Comment:
Just for future, should we use GET, POST, PATCH methods to indicate for fetching, creating, editing in the `/admin/events/ endpoint`?

Reply:
Was also thinking of creating another ticket to address these changes as a few endpoints have some endpoints that could use the three methods instead of explicitly stating "create".

Reply:
👍

Reply:
yeah actually probably a good idea to do it now

Reply:
we should write a wiki article enforcing this?